bom/runtime/pom.xml

```diff
@@ -1096,11 +1089,6 @@
                 <artifactId>javax.ws.rs-api</artifactId>
                 <version>${javax.ws.rs-api.version}</version>
             </dependency>
-            <dependency>
```

Comment:
Have you run this with TCKs enabled?

I suspect removing the junit dependency will break them

Reply:
I browsed the POMs, and noticed they depend on `quarkus-arquillian`, which transitively resolves JUnit 4 through `arquillian-junit-container`, but I'll double check

Reply:
The `common` module has a dependency on junit.

It might come in through Arq BOM

Reply:
Not only that, but the tests in other modules (including the core) depend on JUnit4. I'll make sure to migrate them as well